README.rst

```diff
@@ -137,6 +163,50 @@ Or, assuming you have tox installed (via :code:`sudo pip install tox`)::
 
     tox
```

Comment:
looks like there is a call to mesos for state... however the json decoder is failing instead of the expected unreachable url